keyboards/drewkeys/iskar/info.json

```diff
@@ -0,0 +1,383 @@
+                {"label": "e 1,4", "x": 3.5, "y": 1},
+                {"label": "r 1,5", "x": 4.5, "y": 1},
+                {"label": "t 1,6", "x": 5.5, "y": 1},
+                {"label": "y 1,7", "x": 6.5, "y": 1},
+                {"label": "u 1,8", "x": 7.5, "y": 1},
+                {"label": "i 1,9", "x": 8.5, "y": 1},
+                {"label": "o 1,10", "x": 9.5, "y": 1},
+                {"label": "p 1,11", "x": 10.5, "y": 1},
+                {"label": "[ 1,12", "x": 11.5, "y": 1},
+                {"label": "] 1,13", "x": 12.5, "y": 1},
+                {"label": "| 1,14", "x": 13.5, "y": 1, "w": 1.5},
+                {"label": "home 1,15", "x": 15, "y": 1},
+
+                {"label": "caps 2,0", "x": 0, "y": 2, "w": 1.75},
+                {"label": "a 2,2", "x": 1.75, "y": 2},
+                {"label": "s 2,3", "x": 2.75, "y": 2},
+                {"label": "d 2,4", "x": 3.75, "y": 2},
+                {"label": "f 2,5", "x": 4.75, "y": 2},
+                {"label": "g 2,6", "x": 5.75, "y": 2},
+                {"label": "h 2,7", "x": 6.75, "y": 2},
+                {"label": "j 2,8", "x": 7.75, "y": 2},
+                {"label": "k 2,9", "x": 8.75, "y": 2},
+                {"label": "l 2,10", "x": 9.75, "y": 2},
+                {"label": "; 2,11", "x": 10.75, "y": 2},
+                {"label": "' 2,12", "x": 11.75, "y": 2},
+                {"label": "enter 2,13", "x": 12.75, "y": 2, "w": 2.25},
+                {"label": "pg up 2,15", "x": 15, "y": 2},
+
+                {"label": "shift 3,1", "x": 0, "y": 3, "w": 2.25},
+                {"label": "z 3,2", "x": 2.25, "y": 3},
+                {"label": "x 3,3", "x": 3.25, "y": 3},
+                {"label": "c 3,4", "x": 4.25, "y": 3},
+                {"label": "v 3,5", "x": 5.25, "y": 3},
+                {"label": "b 3,6", "x": 6.25, "y": 3},
+                {"label": "n 3,7", "x": 7.25, "y": 3},
+                {"label": "m 3,8", "x": 8.25, "y": 3},
+                {"label": ", 3,9", "x": 9.25, "y": 3},
+                {"label": ". 3,10", "x": 10.25, "y": 3},
+                {"label": "/ 3,11", "x": 11.25, "y": 3},
+                {"label": "shift 3,13", "x": 12.25, "y":3, "w": 1.75},
+                {"label": "up 3,14", "x": 14, "y": 3},
+                {"label": "pg dn 3,15", "x": 15, "y": 3},
+
+                {"label": "ctrl 4,0", "x": 0, "y": 4, "w": 1.25},
+                {"label": "win 4,1", "x": 1.25, "y": 4, "w": 1.25},
+                {"label": "alt 4,2", "x": 2.5, "y": 4, "w": 1.25},
+                {"label": "spacebar 4,6", "x": 3.75, "y": 4, "w": 6.25},
+                {"label": "fn 4,10", "x": 10, "y": 4, "w": 1.25},
+                {"label": "ctrl 4,11", "x": 11.25, "y": 4, "w": 1.25},
+                {"label": "left 4,13", "x": 13, "y": 4},
+                {"label": "down 4,14", "x": 14, "y": 4},
+                {"label": "right 4,15", "x": 15, "y": 4}
+            ]
+        },
+        "LAYOUT_65_ansi": {
```

Comment:
These layouts don't exist in the .h file.

Reply:
Apologies, I thought as long as the same physical switches are used inside info.json it would not have to be explicitly defined in the <keyboard>.h file. Thanks!